.github/workflows/helm_chart_release.yaml

```diff
@@ -20,6 +20,9 @@ jobs:
         run: |
           git config user.name "$GITHUB_ACTOR"
           git config user.email "$GITHUB_ACTOR@users.noreply.github.com"
+
+      - name: Install bom
+        uses: kubernetes-sigs/release-actions/setup-bom@dd08496c83441d6477114cc0555b96d404dacff7 # v0.1.2
```

Comment:
Awesome that this exists!

This release workflow is only for the helm chart, we haven't figured out the right way to automate releases for the primary artifacts of this project (`cloud-controller-manager-aws` and `ecr-credential-provider`). The binaries and container images are built after a new tag is created, and need to be promoted to the prod registry via 2 PR's, examples:
- https://github.com/kubernetes/k8s.io/pull/6195
- https://github.com/kubernetes/k8s.io/pull/6111

We shouldn't create a GitHub Release until after those PR's merge (in my opinion). And the SBOM should be attached to that release, not the helm chart's (IIUC). If we could:
1. Create a blank GitHub Release for an existing tag after image+artifact promo is complete.
2. Trigger a workflow that would update the Release with the SBOM and references to the prod images/artifacts.

Reply:
I see, sorry about this, I don't have the full context around where the artifacts are built. Do you think it would be better/possible if it could be added to  `cloudbuild.yaml`, or where the artifacts are built instead of adding them later?

Reply:
If we could generate the SBOM in the cloudbuild job and include that in the artifacts, that'd be great!